.craft.yml

```diff
@@ -1,9 +1,14 @@
 ---
+minVersion: "0.14.0"
 github:
   owner: getsentry
   repo: sentry-netlify-build-plugin
 changelogPolicy: simple
```

Comment:
You may also consider `auto`: https://github.com/getsentry/craft#changelog-policies